go-controller/pkg/ovn/hybrid_test.go

```diff
@@ -1243,6 +1243,7 @@ var _ = ginkgo.Describe("Hybrid SDN Master Operations", func() {
 				DnatSnatIP:           "169.254.0.1",
 			}
 			testNode := node1.k8sNode("2")
+			testNode.Annotations[hotypes.HybridOverlayDRIP] = nodeHOIP
```

Comment:
why this change?

Reply:
I think you did this to try to make the unit test pass, but it is flawed anyway. Will be fixed by https://github.com/ovn-org/ovn-kubernetes/pull/4583/files

Remove this change and the rest of your PR lgtm